src/NuGet.Core/NuGet.Packaging/PublicAPI/netcoreapp5.0/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,6 @@
 #nullable enable
+NuGet.Packaging.Rules.MissingReadmeRule
+~NuGet.Packaging.Rules.MissingReadmeRule.MessageFormat.get -> string
+~NuGet.Packaging.Rules.MissingReadmeRule.MissingReadmeRule(string messageFormat) -> void
+~NuGet.Packaging.Rules.MissingReadmeRule.Validate(NuGet.Packaging.PackageArchiveReader builder) -> System.Collections.Generic.IEnumerable<NuGet.Common.PackagingLogMessage>
+~static NuGet.Packaging.Rules.AnalysisResources.MissingReadmeInformation.get -> string
```

Comment:
I think it would be great if we start adding nullable annotations to our new public APIs, and we'll get the added benefit of having the compiler check nulls in our code as well. It's not always easy to enable nullable checks on old code, but I think it would be great, and reduce bugs, if we start enabling nullable on new files.